### PR TITLE
Validation update

### PR DESCRIPTION
**Pull request by jthorton**
## Management update
This PR reworks some of the current validation workflow to work with the dataset update in the newest version of qcsubmit.

### CODE REVIEW

File: management/validation.py [hunk 1]
```diff
@@ -179,14 +185,16 @@ def check_qcspec_coverage(dataset_data):
     # now try and add each spec
     spec_report = {}
     for spec in qc_specs.values():
-
+        valid_scf_props = check_scf_props(spec)
         try:
+            # remove the scf props so they dont cause issues
+            del spec["scf_properties"]
```
Comment (dotsdl) [inline management/validation.py:191]: What kind of issues? I don't understand why we would delete the `"scf_properties"` key from the spec dict here.

**Review by dotsdl — APPROVED** [review]
Thanks @jthorton, sorry for the delay here. I have one item I don't understand. Can you clarify?

File: management/validation.py (hunk 1 above)
Comment (pavankum) [reply management/validation.py:191]: just curious about this, I am hitting validation errors on my new submission #220

File: management/validation.py [hunk 2]
```diff
@@ -179,14 +185,16 @@ def check_qcspec_coverage(dataset_data):
     # now try and add each spec
     spec_report = {}
     for spec in qc_specs.values():
-
+        valid_scf_props = check_scf_props(spec)
```
Comment (jthorton) [inline management/validation.py:188]: @dotsdl  @pavankum We now validate the scf properties separately to the whole QC specification in this line.

File: management/validation.py (hunk 1 above)
Comment (jthorton) [reply management/validation.py:191]: My thinking was as the scf properties are now validated separately on line 188 if they are incorrect they should not then raise an error when validating the method, basis and program so I remove them. The current errors in PRs #220 and #223 are due to the qcsubmit dataset changes which moved the `scf_properties` from the dataset to the qcspecifications.

Comment (dotsdl) [reply management/validation.py:191]: Ah, understood. Thank you! I'll add these details to the inline comment.

**Comment by dotsdl** [comment]
Merged! Thanks @jthorton! Apologies again for the delay on my part.

**Pull request merged**